rotkehlchen/tests/unit/decoders/test_1inch.py

```diff
@@ -163,6 +168,65 @@ def test_1inchv2_swap_for_eth(database, ethereum_inquirer):
     assert expected_events == events
 
 
+@pytest.mark.parametrize('ethereum_accounts', [['0x312419eEC9C4632155904D9440dc1EeeafFBb280']])
+def test_1inchv4_swap_on_uniswapv3(database, ethereum_inquirer):
```

Comment:
I would need @yabirgb to also check this fix work for the thing he wanted to fix.

Reply:
yes, I got my transaction